src/wagtail_ai/models.py

```diff
@@ -0,0 +1,106 @@
+import uuid
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from wagtail.search import index
+
+DEFAULT_PROMPTS = [
+    {
+        "uuid": "fe029b02-833e-49d6-8002-14619962946a",  # This is a static UUID used to identify the prompt
```

Comment:
**Suggestion:** We could store those as `uuid.UUID` objects so we don't have to cast them to strings for comparisons.